src/plugins/team-goal-mixin.js

```diff
@@ -48,10 +48,17 @@ export default {
 		daysLeft() {
 			const start = this.goal?.startDate ? new Date(this.goal?.startDate) : new Date();
 			const end = this.goal?.endDate ? new Date(this.goal?.endDate) : new Date();
-			return intervalToDuration({
+			const interval = intervalToDuration({
 				start,
 				end,
-			}).days;
+			});
+
+			if (interval?.months) {
+				const daysInMonth = getDaysInMonth(start);
+				return interval.days + (interval.months * daysInMonth);
```

Comment:
Not sure if we should cover this right now, but what happens if we have more than 1 month?